Do not use -Werror. Fixes build failure on Mac OS X 10.7 and 10.8:

error: use of logical '&&' with constant operand [-Werror,-Wconstant-logical-operand]
--- setup.py.orig	2020-10-30 07:52:50.000000000 -0500
+++ setup.py	2020-11-12 13:04:08.000000000 -0600
@@ -434,7 +434,7 @@
         extra_compile_args.append("/Z7")
         extra_link_args.append("/DEBUG")
 else:
-    extra_compile_args = ["-Werror"]
+    extra_compile_args = []
     extra_link_args = []
     if debugging_symbols_requested:
         extra_compile_args.append("-g")
